pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -0,0 +1,2778 @@
+/*
```

Comment:
@yifan-gu See `hack/update-generated-runtime.sh` and `hack/verify-generated-runtime.sh`